packages/advanced-logic/src/extensions/payment-network/any-to-erc20-proxy.ts

```diff
@@ -9,7 +10,12 @@ export default class AnyToErc20ProxyPaymentNetwork extends Erc20FeeProxyPaymentN
     extensionId: ExtensionTypes.ID = ExtensionTypes.ID.PAYMENT_NETWORK_ANY_TO_ERC20_PROXY,
     currentVersion: string = CURRENT_VERSION,
   ) {
-    super(extensionId, currentVersion, [], RequestLogicTypes.CURRENCY.ERC20);
```

Comment:
Here the supported networks is an empty array. I think it's an error but please correct me if I'm wrong. The default `isValidAddress()` implementation doesn't work without changing this.

Reply:
Good move !